src/views/docs/en/guides/developer-experience/local-development.md

```diff
@@ -4,112 +4,103 @@ category: Developer experience
 description: How to develop locally with Architect sandbox
 ---
 
-Fast local development creates a tighter feedback loop maximizing developer velocity.
+Fast local development creates a tighter feedback loop, maximizing developer velocity. Architect provides utilities to run projects locally, provision new resources, and debug your application.
 
-## Preview
+## Running locally
 
-Follow the [quickstart](/docs/en/guides/get-started/quickstart) to get everything wired up. To preview a project running locally in a web browser:
+> 🏁 To set up Architect locally follow the [quickstart guide](/docs/en/guides/get-started/quickstart).
 
-```bash
+Preview a project running locally in a web browser by starting Architect's Sandbox:
+
+```console
 cd myproject
 arc sandbox
 ```
->  `arc sandbox` kicks up a local web server with all the resources defined in your `app.arc` file
-## Testing
+
+>  [`arc sandbox`](/docs/en/reference/cli/sandbox) spins up a local web server with all the resources defined in manifest, emulating a suite of AWS features.
 
 Checkout [a complete example project for working locally](https://github.com/architect-examples/arc-example-working-locally).
 
-### Testing `@http` functions
+## Creating a new resource
+
+You can create a new resource (HTTP route, event, scheduled task, etc.) to an existing project by simply adding a line to the [Architect manifest file](/docs/en/guides/get-started/project-layout#manifest-file-format-overview). The following example will provision a new "hit counter" event.
+
+> 👉 Note: this example uses Node.js conventions but the process is similar for the [Ruby](/docs/en/reference/runtime/ruby) (bundler) and [Python](/docs/en/reference/runtime/python) (pip) runtimes.
+
+### Update Architect's configuration
+
+To create a new event, add an entry to your manifest's [`@events` pragma](/docs/en/reference/app.arc/events).
+
+Sample `app.arc` file with a new `hit-counter` event:
+
+```arc
+# app.arc
+@app
+my-app
+
+@events
+hit-counter
+```
+
+> 💡 If the pragma does not already exist, add "@events" on a new line.
+
+### Scaffold the new resource with `arc init`
+
+The Architect CLI [`init` command](/docs/en/reference/cli/init) can be used to create scaffolding for the new event. From the project root:
 
-Use `@architect/sandbox` as a module to test `@http` functions with Node:
+```console
+arc init
+```
+
+> 👀 Notice that `arc` finds your updated Architect manifest and creates new project files for the hit-counter event. `arc init` can be run as needed while developing your application.
 
-```javascript
-const test = require('tape')
-const tiny = require('tiny-json-http')
-const sandbox = require('@architect/sandbox')
+The following structure was added your project:
+<!-- unsure if this diagram is necessary -->
+```
+.
+├── src
+│   └── events
+│       └── hit-counter
+│           ├── index.js
+│           └── config.arc
+│
+└── app.arc
```

Comment:
Hmm, some funky unicode artifacts here?
![image](https://user-images.githubusercontent.com/200964/128097791-4d618aaa-d70f-4144-8034-6a156d46b928.png)

Reply:
Super weird. that line is the same as the other `└── app.arc` instances in project-layout.md and queues.md.  
and the `─` character that shows up garbled in your screenshot is used in each of the lines above the app.arc line in your screenshot.

The issue must be between the code and the HTML output, maybe the Markdown compiler. but I'm having a hard time reproducing that garbled text.  
I'm already taking a look there since there seems to be an issue with my highlighter.js change, so I can take some time to look more at this.